helm/cert-operator-chart/templates/deployment.yaml

```diff
@@ -54,6 +54,12 @@ spec:
         - --config.dirs=/var/run/cert-operator/secret/
         - --config.files=config
         - --config.files=secret
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8000
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
```

Comment:
I think it would make sense to also define the readiness probe like in other deployments. Keeping the templates in sync will make it easier to unify or abstract them later, or at least to understand them.